src/Logger/Log.php

```diff
@@ -302,7 +302,7 @@ public function getTags(): array
      * @param $value
      * @return void
      */
-    public function addExtra(string $key, $value): void
+    public function addExtra(string $key, $value): void // @phpstan-ignore-line
```

Comment:
Please add a type rather than ignore:

```suggestion
    public function addExtra(string $key, mixed $value): void
```